src/coreclr/src/vm/pefile.cpp

```diff
@@ -2331,6 +2335,24 @@ void PEFile::EnsureImageOpened()
         GetILimage()->GetLayout(PEImageLayout::LAYOUT_ANY,PEImage::LAYOUT_CREATEIFNEEDED)->Release();
 }
 
+AssemblyLoadContext* PEFile::LookupAssemblyLoadContext()
```

Comment:
`SetAssemblyLoadContext` + `LookupAssemblyLoadContext` can be merged into a single method like `void SetupAssemblyLoadContext()` that computes it from the other PEFile state and sets it.